### PR TITLE
Added binary test for "older iphones"

### PR DESCRIPTION
**Pull request by gklopper**
Switching off half the "features" to see if the crash metric moves. After that will follow a simple binary diff.

### CODE REVIEW

**Comment by gklopper** [comment]
cc @phamann

**Comment by phamann** [comment]
:+1: Yepp looks good to me.

File: common/app/views/fragments/javaScriptFirstSteps.scala.html [hunk 1]
```diff
@@ -194,9 +194,19 @@
 
 </script>
 
-@* Not the usual type of thing we do, but I want this separate to our "normal" javascript. *@
-@* also delete iPhoneConfidenceCheck inside of facia.js *@
+
+<script>
+    guardian.isIphone6 = false;
+    guardian.isIphone4 = false;
+    guardian.inTestBucket = Math.floor(Math.random() * 2 + 1) === 1;
```
Comment (piuccio) [inline common/app/views/fragments/javaScriptFirstSteps.scala.html]: I know that it's been merged already, but this line is quite hard to read.

For the future i'd suggest something like
`guardian.inTestBucket = Math.random() < 0.5;`
It's easier to understand the ratio

Comment (gklopper) [inline common/app/views/fragments/javaScriptFirstSteps.scala.html]: True, lots of this code is hacky. My only defence is its just some debug code and will be deleted as soon as we get to the bottom of the crashing problem :frowning:

**Pull request merged**